api/lambdas/src/token-auth.ts

```diff
@@ -0,0 +1,80 @@
+import * as Sentry from "@sentry/serverless";
+import * as lambda from "aws-lambda";
+import { DynamoDB } from "aws-sdk";
+import { capture } from "./shared/capture";
+import { getEnvOrFail } from "./shared/env";
+
+// Keep this as early on the file as possible
+capture.init();
+
+const tableName = getEnvOrFail("TOKEN_TABLE_NAME");
```

Comment:
Previously, this was just a `process.env.TOKEN_TABLE_NAME` on the code below:

```js
      const doc = await docClient
        .get({
          TableName: process.env.TOKEN_TABLE_NAME,
          Key: { token: token },
        })
        .promise();
```

Thought this is better, but happy to review.